src/databricks/labs/ucx/source_code/notebook.py

```diff
@@ -225,6 +292,41 @@ def make_cell(lines_: list[str]):
 
         return cells
 
+    def _make_runnable(self, lines: list[str], cell_language: CellLanguage):
```

Comment:
```suggestion
    def _unwrap_magic(self, lines: list[str], cell_language: CellLanguage):
```
i think it's a better name for this

Reply:
done